### PR TITLE
[AssemblyScript] Add unrolled solution

### PR DESCRIPTION
**Pull request by MaxGraey**
## Description

This is unrolled bit set solution.

* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

**Comment by rbergen** [comment]
@MaxGraey Could you please add this as a second implementation to solution_2?

**Comment by MaxGraey** [comment]
Sure!

**Comment by MaxGraey** [comment]
@rbergen done!

File: PrimeAssemblyScript/solution_2/Dockerfile [hunk 1]
```diff
@@ -7,6 +7,8 @@ RUN npm install
 COPY . .
 
 
-RUN npm run build:bench
+RUN npm run build:orig
+RUN npm run build:unrolled
 
-CMD [ "npm", "run", "bench" ]
+CMD [ "npm", "run", "bench:orig" ]
+CMD [ "npm", "run", "bench:unrolled" ]
```
Comment (rbergen) [inline PrimeAssemblyScript/solution_2/Dockerfile]: As the lint warning explains, adding multiple CMD instructions will not actually run multiple solutions, but only the last one listed. An approach that is often used to run multiple solutions from one Dockerfile is to write a small wrapper script (commonly called "run.sh") that executes the `npm run` commands, in this case.

Comment (MaxGraey) [reply PrimeAssemblyScript/solution_2/Dockerfile]: Got it!

Comment (MaxGraey) [reply PrimeAssemblyScript/solution_2/Dockerfile]: I added new aggregated command "bench:all" to package.json. And call it in dockerfile

**Review by rbergen — APPROVED** [review]
LGTM

**Review by marghidanu — APPROVED** [review]

**Pull request merged**